src/docs/components/docs-page-nav/docs-page-nav.tsx

```diff
@@ -40,8 +40,9 @@ export class DocsPageNav {
           <div class="docs-page-nav__content">
             {this.prevHref ? (
               <ld-button
+                brand-color
```

Comment:
Shouldn't we call this `on-brand-color` to make clear that the button is used _on_ a brand color background? Otherwise one could misinterpret this as the button has brand color.

Reply:
Yes, we should. But sorrily this is not possible with Stencil, as it expects props starting with `on` to be event handlers.